bitnami/mediawiki/templates/NOTES.txt

```diff
@@ -22,8 +22,8 @@ host. To configure Mediawiki with the URL of your service:
 
   export APP_HOST=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "common.names.fullname" . }} --template "{{ "{{ range (index .status.loadBalancer.ingress 0) }}{{ . }}{{ end }}" }}")
   export APP_PASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "common.names.fullname" . }} -o jsonpath="{.data.mediawiki-password}" | base64 --decode)
-  export MARIADB_ROOT_PASSWORD=$(kubectl get secret --namespace default mediawiki-mariadb -o jsonpath="{.data.mariadb-root-password}" | base64 --decode)  {{- end }}
-  export MARIADB_PASSWORD=$(kubectl get secret --namespace default mediawiki-mariadb -o jsonpath="{.data.mariadb-password}" | base64 --decode)
+  export MARIADB_ROOT_PASSWORD=$(kubectl get secret --namespace default {{ .Release.Namespace }}-mariadb -o jsonpath="{.data.mariadb-root-password}" | base64 --decode)  {{- end }}
+  export MARIADB_PASSWORD=$(kubectl get secret --namespace default {{ .Release.Namespace }}-mariadb -o jsonpath="{.data.mariadb-password}" | base64 --decode)
```

Comment:
Thanks for your contribution. A couple of comments:
- `--namespace` should be generated dynamically, if not, when used a different namespace than the default one it is not going to work
- The secret name is not generated using the namespace but using the release name, for that purpose there is a template defined in the helpers.tpl:
```
{{/*
Create a default fully qualified app name.
We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
*/}}
{{- define "mediawiki.mariadb.fullname" -}}
{{- printf "%s-%s" .Release.Name "mariadb" | trunc 63 | trimSuffix "-" -}}
{{- end -}}
```

```suggestion
  export MARIADB_ROOT_PASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "mediawiki.mariadb.fullname" . }} -o jsonpath="{.data.mariadb-root-password}" | base64 --decode)  {{- end }}
  export MARIADB_PASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "mediawiki.mariadb.fullname" . }} -o jsonpath="{.data.mariadb-password}" | base64 --decode)
```